.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
   - if [ "$DEPENDENCIES" == "low" ]; then composer update --prefer-lowest; fi;
 
 before_script:
-  - echo "<?php if (PHP_VERSION_ID >= 50400) echo ',@php5.4';" > php_version_tags.php
```

Comment:
Any suggestions? I don't have any ideas. This looks a little like a hack, but it is a small and easy to maintain hack :smile: